screens/Article.jsx

```diff
@@ -32,9 +33,6 @@ export default function Article({ route }) {
 		width: `${scrollPercentage}%`,
 	};
 
-	useEffect(() => {
-	}, []);
-
```

Comment:
are we 100% sure that removing this is on purpose

Reply:
Added it back, no changes, will keep it added tho